bower.json

```diff
@@ -1,9 +1,8 @@
 {
-  "name": "KaTeX",
-  "version": "0.8.0-pre",
+  "name": "katex",
   "main": [
-    "dist/katex.min.js",
-    "dist/katex.min.css"
+    "dist/katex.js",
+    "dist/katex.css"
```

Comment:
Hrm. It's annoying that bower doesn't let us specify several different things with the same filetype. I think people would probably prefer the minified version to be the default? Not sure though.